packages/eslint-config-nimble-testing/README.md

```diff
@@ -0,0 +1,66 @@
+# `@nimblehq/eslint-config-nimble-testing`
+
+This package provides ESLint testing rules inheriting from @nimblehq/eslint-config-nimble-core.
+
+## Caution
+@nimblehq/eslint-config-nimble-core includes `jest` and `cypress` as the primary test suite stack. Consider if the targeting project is not using those libraries.
+
+## Installation
+
+```bash
+  npm install --save-dev @nimblehq/eslint-config-nimble-testing
+```
+
+## Usage
+
+### Standalone
+
+Add `@nimblehq/eslint-config-nimble-testing` to the extends section of your `.eslintrc` configuration file.
+
+```js
+{
+  "extends": [
+      "@nimblehq/eslint-config-nimble-testing"
+  ],
+  "rules": {
+    // Additional, per-project rules...
+  }
+}
+```
+
+### With a framework
+
+Similar to the process above, but usually it requires adding the extra rules for the JS framework:
+
+```js
+{
+  "extends": [
+      "@nimblehq/eslint-config-nimble-testing",
+      "plugin:react/recommended",
+      "plugin:vue/recommended"
+  ],
+  "rules": {
+    // Additional, per-project rules...
+  }
+}
+```
+
+This would require defining the required dependencies in the project itself.
+
+## License
+
+This project is Copyright (c) 2014 and onwards Nimble. It is free software and may be redistributed under the terms specified in the [LICENSE] file.
```

Comment:
```suggestion
This project is Copyright (c) 2021 and onwards Nimble. It is free software and may be redistributed under the terms specified in the [LICENSE] file.
```

That's a very minor detail but the LICENSE file says `2021 and onwards` so should we have the same here? ^^

Reply:
Nice catch, updated all in [1af4364](https://github.com/nimblehq/eslint-config-nimble/pull/49/commits/1af43648757c88b92f31a824419b206ce635d8fa)